docs/DeveloperGuide.adoc

```diff
@@ -154,77 +154,79 @@ The sections below give more details of each component.
 
 [[Design-Ui]]
 === UI component
-
-.Structure of the UI Component
-image::UiClassDiagram.png[width="800"]
-
-*API* : link:{repoURL}/src/main/java/seedu/address/ui/Ui.java[`Ui.java`]
+*API* : link:{repoURL}/src/main/java/seedu/club/ui/Ui.java[`Ui.java`]
 
 The UI consists of a `MainWindow` that is made up of parts e.g.`CommandBox`, `ResultDisplay`, `PersonListPanel`, `StatusBarFooter`, `BrowserPanel` etc. All these, including the `MainWindow`, inherit from the abstract `UiPart` class.
```

Comment:
MemberListPanel instead of PersonListPanel

Reply:
Should we rename the diagrams to member instead of person also? Eg DeletePersonSdForLogic.png to DeleteMemberSdForLogic.png

Reply:
https://github.com/CS2103JAN2018-W15-B4/main/pull/96#discussion_r175293424  👍

Reply:
https://github.com/CS2103JAN2018-W15-B4/main/pull/96#discussion_r175293572 Can leave this for a later PR? 😅 we can do all of them at once